ui/plugins/failure_summary/controller.js

```diff
@@ -46,6 +48,16 @@ treeherder.controller('BugsPluginCtrl', [
                     if(artifact_list.length > 0){
                         var artifact = _.find(artifact_list, {"name": "Bug suggestions"});
                         angular.forEach(artifact.blob, function (suggestion) {
+                            // Build up lists of bug ids for use when we're over bug_limit
+                            suggestion.bugs.list_recent = "";
+                            suggestion.bugs.list_others = "";
+                            angular.forEach(suggestion.bugs.open_recent, function(bug) {
+                                suggestion.bugs.list_recent += bug.id + ",";
```

Comment:
Rather than duplicating this, we should be able to just display using a filter. Or alternatively, we could just display the same as we normally do (ie bug summary too), and just require the user to have first pressed a "show anyway" link.